internal/services/logic/integration_service_environment.go

```diff
@@ -28,10 +28,11 @@ import (
 
 func resourceIntegrationServiceEnvironment() *pluginsdk.Resource {
 	return &pluginsdk.Resource{
-		Create: resourceIntegrationServiceEnvironmentCreateUpdate,
-		Read:   resourceIntegrationServiceEnvironmentRead,
-		Update: resourceIntegrationServiceEnvironmentCreateUpdate,
-		Delete: resourceIntegrationServiceEnvironmentDelete,
+		DeprecationMessage: "Integration service environments are getting deprecated. Please use Logic Apps Standard instead. Please see https://aka.ms/isedeprecation for more details.",
```

Comment:
can we explain this a little more?

```suggestion
		DeprecationMessage: `The "azurerm_integrated_service_environment" resource is deprecated and will be removed in v4.0 of the Azure Provider. The underlying Azure Service is being retired on 2024-08-31 and new instances cannot be provisioned by default after 2022-11-01. More information on the retirement and how to migrate to Logic Apps Standard ("azurerm_logic_app_standard") can be found here: https://aka.ms/isedeprecation.
```